wallet/src/actors/app/methods.rs

```diff
@@ -624,17 +624,37 @@ impl App {
         Box::new(f)
     }
 
+    /// Send a transaction to the node as inventory item broadcast
+    /// and add a local pending balance movement to the wallet state.
     pub fn send_transaction(
         &self,
         session_id: &types::SessionId,
         wallet_id: &str,
         transaction: types::Transaction,
-    ) -> ResponseActFuture<serde_json::Value> {
+    ) -> ResponseActFuture<SendTransactionResponse> {
```

Comment:
Does this change break the public API? (sheikah)

Reply:
Yes, but I think nobody was using it...the response was the json rpc value of sendinng an InventoryItem

Reply:
Yes, the response was the transaction hash. Now it's an object that contains the transaction hash and a `BalanceMovement`, which is some data structure that's internal to the wallet. I think it would be better to keep this method as before. Can we just map the result like `.map(|result| result.jsonrpc_result)` ?

Reply:
They can use the `BalanceMovement` object to draw a new pending transaction in their transaction list. :smiley:

Reply:
Ok, then leave it like this.